core/src/main/antlr4/org/jdbi/v3/core/internal/lexer/ColonStatementLexer.g4

```diff
@@ -27,7 +27,7 @@ fragment NAME: JAVA_LETTER | [0-9] | '.' | '?.';
 /* Lovingly lifted from https://github.com/antlr/grammars-v4/blob/master/java/JavaLexer.g4 */
 fragment JAVA_LETTER : [a-zA-Z$_] | ~[\u0000-\u007F\uD800-\uDBFF] | [\uD800-\uDBFF] [\uDC00-\uDFFF];
 
-COMMENT: '/*' .*? '*/';
+COMMENT: '/*' .*? '*/' | '--' ~('\r' | '\n')* | '//' ~('\n' | '\r')*;
```

Comment:
Is there any particular reasoning behind the different order of \\r and \\n in the two sides here, or just coincidence? If no reason, I'd prefer both in \\r\\n order (and the same in the other files of course)

Reply:
Hell no, thanks for catching this. I've also aligned newline patterns in `SqlScriptLexer.g4` to use \r\n order, which is otherwise unchanged.